### PR TITLE
docs(beginner tutorial unit testing): revise prose and bug-fix

### LINKED ISSUE #3360: [Documentation] Beginner Tutorial Utilities Unit Testing Bug
#### Existing page / new
[Lesson 1.4](https://www.evennia.com/docs/latest/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.html#testing)

`BaseEvenniaTest` will use the default evennia typeclasses rather than EvAdventure's

#### Documentation issue
```python
from evennia.utils.test_resources import BaseEvenniaTest 

...

class TestUtils(BaseEvenniaTest):
```
#### Suggested change
```python
from evennia.utils.test_resources import EvenniaTest 

...

class TestUtils(EvenniaTest):
```

### PR DESCRIPTION
**Pull request by iLPdev**
#### Brief overview of PR changes/additions
General prose editing for punctuation, syntax, grammar, clarity, etc. including fix for #3360.    

#### Motivation for adding to Evennia
progress, not perfection

#### Other info (issues closed, discussion etc)
closes #3360

### CODE REVIEW

File: docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.md [hunk 1]
```diff
@@ -87,15 +86,15 @@ Ability.STR.value  # this is the string "strength"
 
 ```
 
-Having enums is recommended practice. With them set up, it means we can make sure to refer to the  same thing every time. Having all enums in one place also means you have a good overview of the  constants you are dealing with.
+Using enums is a recommended practice. With enums set up, we can make sure to refer to the same contant or variable every time. Keeping all enums in one place also means we have a good overview of the constants with which we are dealing.
```
Comment (Griatch) [inline docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.md:89]: contant->constant

File: docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.md [hunk 2]
```diff
@@ -212,43 +211,42 @@ def get_obj_stats(obj, owner=None):
...
-We can guess some things, such that `obj.key` is the name of the object, and that `obj.db.desc` will  hold its description (this is how it is in default Evennia).
+Some stats elements are easy to identify in the above code. For instance, `obj.key` is the name of an object and `obj.db.desc` will hold an object's description &mdash; this is also how default Evennia works.
 
-But so far we have not established how to get any of the other properties like `size` or `attack_type`. So we just set them to dummy values. We'll need to get back to this when we have more code in place!
+So far, here in our tutorial, we have not yet established how to get any of the other properties like `size` or `attack_type`. For our current purposes, we will just set them to dummy values and we'll need to revisit them later when we have more code in place!
 
 ## Testing 
 
+Evennia comes with extensive functionality to help you test your code. A _unit test_ allows you to set up automated testing of code. Once you've written your test, you can than run it over and over again to ensure later changes to your code didn't break things by introducing errors. 
+
```
Comment (Griatch) [inline docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.md:223]: than->then

File: docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.md [hunk 3]
```diff
@@ -212,43 +211,42 @@ def get_obj_stats(obj, owner=None):
...
 > create a new module `mygame/evadventure/tests/test_utils.py`
 
-How do you know if you made a typo in the code above? You could _manually_ test it by reloading your  Evennia server and do the following from in-game: 
+How would you know if you made a typo in the code above? You can _manually_ test it by reloading your Evennia server and issuing the following in-game python command: 
 
     py from evadventure.utils import get_obj_stats;print(get_obj_stats(self))
 
-You should get back a nice string about yourself! If that works, great! But you'll need to remember doing that test when you change this code later. 
+Doing so should spit back a nice bit of string ouput about yourself! If that works, great! But, you'll need to remember re-running that test mannually when you later change the code.
```
Comment (Griatch) [inline docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Utilities.md:230]: mannually -> manually

**Review by Griatch — CHANGES_REQUESTED** [review]
Good stuff! Just some remaining spelling errors/typos.

**Comment by iLPdev** [comment]
nice catches!!

**Review by Griatch — CHANGES_REQUESTED** [review]
Thanks for correcting! Seems there is a merge conflict (upstream likely updated slightly and changes a string in the same place); just fix that and we can merge :)

**Comment by Griatch** [comment]
Ping. This just needs a rebase.

**Comment by Griatch** [comment]
Another ping.

**Comment by Griatch** [comment]
Resolved merge conflict and merged this one since it's been hanging around for a long time.

**Pull request merged**